shell/initialize/entry-helpers.js

```diff
@@ -380,59 +121,13 @@ export const setContext = async(app, context) => {
   if (!app.context) {
     app.context = {
       isDev:   true,
-      isHMR:   false,
       app,
       store:   app.store,
       payload: context.payload,
       error:   context.error,
       base:    app.router.options.base,
-      env:     {
-        commit: 'head', version: '0.1.2', dev: true, pl: 1, perfTest: false, rancherEnv: 'web', api: 'http://localhost:8989'
-      }
```

Comment:
These are completely unused and replaced by the usage of DefinePlugin inside vue.config.js.